src/django/api/tests/test_opensearch_relevant_data.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from django.utils import timezone
+from api.services.search import \
+    OpenSearchService
+from api.models import (
+    Contributor,
+    Facility,
+    FacilityList,
+    FacilityListItem,
+    Source,
+    User,
+)
+from django.contrib.gis.geos import Point
+
+
+class TestPrepareOpenSearchResponse(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_client = MagicMock()
+        self.service = OpenSearchService(client=self.mock_client)
+
+        self.email = "user@example.com"
+        self.password = "password"
+        self.user = User.objects.create(email=self.email)
+        self.user.set_password(self.password)
+        self.user.save()
+
+        self.contributor = Contributor(name="Contributor", admin=self.user)
+        self.contributor.save()
+
+        self.list_one = FacilityList.objects.create(
+            header="header", file_name="one", name="list"
+        )
+
+        self.source_one = Source.objects.create(
+            facility_list=self.list_one,
+            source_type=Source.LIST,
+            is_active=True,
+            is_public=True,
+            contributor=self.contributor,
+        )
+
+        self.list_item_one = FacilityListItem.objects.create(
+            name="name",
+            address="address",
+            country_code="US",
+            sector=["Apparel"],
+            source=self.source_one,
+            row_index=1,
+            status=FacilityListItem.CONFIRMED_MATCH,
+        )
+
+        self.facility = Facility.objects.create(
+            name="name",
+            address="address",
+            country_code="US",
+            location=Point(0, 0),
+            created_from=self.list_item_one,
+        )
+
+        self.old_updated_at = '2019-03-24 02:23:22.195 +0100'
+
+        self.facility.updated_at = self.old_updated_at
+        self.facility.save()
+
+    @patch('api.services.search.logger')
+    def test_update_facility_updated_at_field(self, mock_logger):
```

Comment:
There are mulitple cases that should update `updated_at` field, which are listed in your PR description:
1. Claim process: approve, reject, deny, claim
2. Claim details
3. Merge facilities (Dashboard)

I see that this test directly checks `update_facility_updated_at_field` static method, but this test suite doesn't contain foremention cases. Is it possible to test them?

Reply:
I've added tests for checking updated_at filed after approve, reject, deny, merge facilities.